### PR TITLE
Adapt 7933 narrative attribution gets read after all of the items

### PR DESCRIPTION
**Pull request by eleanor-heath**
resolves https://github.com/adaptlearning/adapt_framework/issues/3052

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

File: templates/narrative.hbs [hunk 1]
```diff
@@ -28,6 +26,8 @@
           </div>
           {{/if}}
 
+          {{#if _graphic.alt}}{{{a11y_aria_image _graphic.alt}}}{{/if}}
```
Comment (oliverfoster) [inline templates/narrative.hbs:29]: Could you now put the alt back on the image tag remove the `aria-hidden` and remove this line?

```js
{{#if _graphic.alt}}aria-label="{{a11y_normalize _graphic.alt}}"{{/if}}
```

Comment (eleanor-heath) [reply templates/narrative.hbs]: It worked! thanks. Can you take another look when you have the chance please @oliverfoster

File: templates/narrative.hbs [hunk 2]
```diff
@@ -81,20 +79,12 @@
...
-        <div class="icon"></div>
-      </button>
-
       <div class="narrative__slider u-clearfix" style="width: {{_totalWidth}}%">
 
         {{#each _items}}
         <div class="narrative__slider-image-container{{#if _isVisited}} is-visited{{/if}}{{#if _graphic.attribution}} has-attribution{{/if}}" style="width: {{../_itemWidth}}%" data-index="{{@index}}">
 
-          <img class="narrative__slider-image" src="{{_graphic.src}}" aria-hidden="true">
+          <img class="narrative__slider-image" src="{{_graphic.src}}" {{#if _graphic.alt}}aria-label="{{a11y_normalize _graphic.alt}}"{{/if}}>
```
Comment (oliverfoster) [inline templates/narrative.hbs:87]: You still need the else aria-hidden otherwise it'll read the file url, which isn't pretty.

**Comment by oliverfoster** [comment]
I'll endeavour to test this with a SR today. :+1:

**Review by oliverfoster — APPROVED** [review]
It's perfect. Except the `u-clearfix` reading blank twice around the buttons. https://github.com/adaptlearning/adapt_framework/issues/3094

**Comment by moloko** [comment]
@oliverfoster do you think it's worth bumping the FW dependency to whatever-I-end-up-calling-the-current-master? or leave as-is and just include something in the release notes that it 'works best with v5.x'?

**Review by moloko — APPROVED** [review]
code 👓

**Comment by oliverfoster** [comment]
A note or version bump would be fine (but not necessary) depending on which fw everyone is using?

The `u-clearfix` issue predates all of these changes. I only just worked out that it's an issue we can fix. Generally I've not been bothered about random `blank`s really, as they're mostly unavoidable.

**Pull request merged**